### ISSUE TITLE
Unrealistic fish weights

### ISSUE THREAD

**Issue by Randinotte** [MFEh2o/db#100]
@cdassow noticed 3 samples with fish weights that are unrealistic compared to the fish's size: 

CR_FN8_20160719_0830_FN_GonadSurvey.20130509_4: 3886mm, 603g
CR_FN9_20160613_1050_FN_GonadSurvey.20130509_3: 26.5mm, 233.1g
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_2: 26.9mm, 243.2g

This will probably require checking through datasheets, which I believe are with @ctsolomon. Is that true?

**Comment by kaijagahm** [comment]
I think you're right that Chris has the Gonad Survey data sheets. 

This is making me think, though, that the time might be right to take nice high-res photos of those data sheets and stick them on Box so that we don't have to bother Chris every time something like this comes up. Thoughts?

**Comment by kaijagahm** [comment]
Scheduled email to Chris about this for Monday.

**Comment by ctsolomon** [comment]
I can't find any data sheets from CR in the 2016 bluegill binder. Also, CR does not appear in the list of lakes that the crew printed on the backs of these data sheets, which makes me think it may not have been in the set of lakes sampled for the 2016 bluegill survey (although it was for Nikki's ~2013 survey). Also, some of those sample dates (especially the July one) seem kinda late for the bluegill survey - they were focused on the spawning period and I think most or all of the sampling was done before July.

All this makes me wonder whether maybe these fish are actually from some other sampling effort, and got entered with the wrong metadataID. @kaijagahm , can you find out any more about these fish/samples?

**Comment by ctsolomon** [comment]
Oh - and yes to the scans of the data sheets, I am going to get someone at Cary moving on that.

**Comment by kaijagahm** [comment]
@ctsolomon looks like you're right about the metadataID being wrong on these. FISH_INFO doesn't give us much information, but I checked FISH_SAMPLES and the `sampleGroup` column is "UNDERC_monitoring" for all three of these sampleID's. If they were actually from the 2016 gonad survey, we'd expect `sampleGroup` to be 'BLGGonadSurvey_2016'.

Where do we go from here? Would the data sheets for 2016 UNDERC monitoring be in the lab at ND, or somewhere else?

Of course, this also throws into doubt the metadataID classifications for other samples. We should talk about how comprehensive a sweep I should make of the sampleID's.

**Comment by Randinotte** [comment]
With @kaijagahm's heads up about the UNDERC_monitoring sampleGroup I found the datasheets in the 2016 UNDERC folder, monitoring tab. The first one was a typo (386 instead of 3886g), but the other two are correct and match the datasheet. I'm not sure how far we want to track this down, but two other LMB were commented "exceeds board" so maybe the CR fish are just chunky?

CR_FN8_20160719_0830_FN_GonadSurvey.20130509_4: **386**mm, 603g (not 3886g as in DB)
CR_FN9_20160613_1050_FN_GonadSurvey.20130509_3: 26.5mm, 233.1g (no change, matches datasheet)
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_2: 26.9mm, 243.2g (no change, matches datasheet)

**Comment by cdassow** [comment]
Sounds good Randi, CR fish ARE chunky (Camille is fast becoming the expert on that). Their comment though makes it seem like they were too long and not too heavy? 26 mm is not big enough to exceed any of the boards we have, as far as I know which makes me wonder if the lengths are wrong and the weights are fine? I don't know how we'd figure that out though.

**Comment by Randinotte** [comment]
@cdassow It's possible that it says "exceeds bound," as the handwriting is a little unclear. The comment is in the weight column though, not in the length column. Maybe "exceeds bound" makes more sense then.

**Comment by Randinotte** [comment]
@cdassow, sorry for another notification, but another question- when did the fish entry tool come about? Did it exist in 2016 in order to incorrectly assign metadataIDs? If it wasn't around, then I'll look for hand-entry/excel causes.

**Comment by cdassow** [comment]
No worries, it came in 2019

**Comment by ctsolomon** [comment]
@cdassow, would the weights make sense if the lengths on those two fish were actually recorded in cm rather than mm?

**Comment by cdassow** [comment]
Yes they would make sense, and measuring to a fraction of a millimeter does seem unrealistic. Based on the comments that Randi saw it maybe seemed more like a weight issue but I think it could easily be a length problem instead

**Comment by Randinotte** [comment]
From Tuesday meeting 1/5/21: 
I'm going to regress length ~ weight by species and project to look for outliers, presuming that monitoring may be more prone to errors. As a note- only the one datasheet from 2016 fish had the "cm not mm" problem, BUT it seems to be all of them on the sheet: 

(as on datasheet)

CR_FN10_20160613_1050_FN_GonadSurvey.20130509: NFC
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_1: 18.1mm, 94.4g 
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_2: 26.9mm, 243.2g
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_3: 18.5mm, 112.1g
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_4: 31.5mm, "exceeds bound"
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_5: 18.2mm, 116.2g
CR_FN6_20160613_1050_FN_GonadSurvey.20130509_6: 12.5mm, 30.9g
CR_FN12_20160613_1050_FN_GonadSurvey.20130509_1: 12.8mm, 28.5g
CR_FN12_20160613_1050_FN_GonadSurvey.20130509_2: 13.5mm, 36.4g
CR_FN12_20160613_1050_FN_GonadSurvey.20130509_3: mort
CR_FN8_20160613_1050_FN_GonadSurvey.20130509_1: 13mm, 34g
CR_FN11_20160613_1050_FN_GonadSurvey.20130509_1: 36.8mm, "exceeds bound"
CR_FN11_20160613_1050_FN_GonadSurvey.20130509_2: "unknown", mort
CR_FN9_20160613_1050_FN_GonadSurvey.20130509_1: 18.0mm, 50.1g
CR_FN9_20160613_1050_FN_GonadSurvey.20130509_2: 20.1mm, 140.6g
CR_FN9_20160613_1050_FN_GonadSurvey.20130509_3: 26.5mm, 233.1g

**Comment by kaijagahm** [comment]
I think this will end up being the same issue as #149

**Comment by kaijagahm** [comment]
This issue has a script started. See 'gh100_fishyWeights.R' and 'gh100_unrealisticFishWeights.R' in my archive.
